js/cex.js

```diff
@@ -118,10 +119,228 @@ module.exports = class cex extends Exchange {
             'options': {
                 'fetchOHLCVWarning': true,
                 'createMarketBuyOrderRequiresPrice': true,
+                'order': {
+                    'status': {
+                        'c': 'canceled',
+                        'd': 'closed',
+                        'cd': 'closed',
+                        'a': 'open',
+                    },
+                },
             },
         });
     }
 
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'limit': limit,
+            'pair': market.id,
```

Comment:
Can you please help cleanup this file according to the rules as described here? https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

Unfortunately, this cleanup is not automated yet.

Reply:
Ah, no worries, everything else is ok, nevermind.